### PR TITLE
add first version of README

### PR DESCRIPTION
**Pull request by rakutenjeff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
 
 This is the repository for ongoing work done by the [W3C Publishing Community Group](https://www.w3.org/community/publishingcg/).
 
+## Specifications
+
+T.B.D
+
+## COntributing to the Publishing Community Group Repositories
+
+Use the standard fork, branch, and pull request workflow to propose changes to the specification. Please make branch names informative窶巴y including the issue or bug number for example.
```
Comment (iherman) [inline README.md]: Jeff, something went wrong and you mixed in some Chinese characters... I presume you wanted to say 'informatively' :-)

Comment (rakutenjeff) [reply README.md]: Lol, yes will fix. Thanks Ivan

File: README.md [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+
 # Publishing Community Group
 
 This is the repository for ongoing work done by the [W3C Publishing Community Group](https://www.w3.org/community/publishingcg/).
 
+## Specifications
+
+T.B.D
+
+## COntributing to the Publishing Community Group Repositories
```
Comment (mteixeira-wwn) [inline README.md]:
```suggestion
## Contributing to the Publishing Community Group Repositories
```

Comment (rakutenjeff) [reply README.md]: thanks, will fix today

File: README.md (hunk 1 above)
Comment (rakutenjeff) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (rakutenjeff) [reply README.md]: done

**Pull request merged**